WalletWasabi.Tests/UnitTests/AllFeeEstimateTests.cs

```diff
@@ -218,18 +268,37 @@ private static MockRpcClient CreateAndConfigureRpcClient(Func<int, FeeRate> esti
 					}),
 				OnGetPeersInfoAsync = async () =>
 					await Task.FromResult(hasPeersInfo ? new[] { new PeerInfo() } : Array.Empty<PeerInfo>()),
-
-				OnGetMempoolInfoAsync = async () =>
-					await Task.FromResult(new MemPoolInfo
-					{
-						MemPoolMinFee = memPoolMinFee // 1 s/b (default value)
-					}),
+				OnGetMempoolInfoAsync = async () => await Task.FromResult(ParseMemPoolInfo(memPoolMinFee)),
 				OnEstimateSmartFeeAsync = async (target, _) =>
 					await Task.FromResult(new EstimateSmartFeeResponse
 					{
 						Blocks = target,
 						FeeRate = estimator(target)
 					})
 			};
-	}
+
+		private static MemPoolInfo ParseMemPoolInfo(double memPoolMinFee)
+		{
+			var mempoolInfoWithHistogram = File.ReadAllText("./UnitTests/Data/MempoolInfoWithHistogram.json");
+			var jo = JObject.Parse(mempoolInfoWithHistogram);
```

Comment:
For new code, I would suggest use of https://docs.microsoft.com/en-us/dotnet/api/system.text.json?view=net-5.0 so that one day we can get rid of Newtonsoft.JSON dependency.

Reply:
It would be good to get rid of Newtonsoft dependency but it seems it is not so easy to do because Adam and David were trying for a good while and decided to revert it. That's why I prefer to do not mix them and use only one.